azure-pipelines/vs-insertion.yml

```diff
@@ -30,6 +30,15 @@ resources:
       branches:
         include: # trigger as a followup to servicing CI
           - vs*
+  - pipeline: 'MSBuild-ExpPerf'
+    project: 'DevDiv'
+    source: 'MSBuild/MSBuild-ExpPerf'
+    branch: exp/*
+    trigger:
+      branches:
+        include:
+          - exp/*
+          - perf/*
```

Comment:
This should not be added back as it was removed in PR https://github.com/dotnet/msbuild/pull/12257.